src/modules/apps/link.ts

```diff
@@ -29,7 +31,71 @@ const UPDATE_SIGN = chalk.blue('U')
 const stabilityThreshold = process.platform === 'darwin' ? 100 : 200
 const AVAILABILITY_TIMEOUT = 1000
 const N_HOSTS = 3
+const builderHubInjectedDepsTimeout = 2000  // 2 seconds
+const yarn = join(__dirname, '../../../node_modules/yarn/bin/yarn.js install --force')
+
+const resolvePackageJsonPath = (builder: string) => resolvePath(process.cwd(), `${builder}/package.json`)
+
+const resolveAppId = async (appName: string, appVersion: string) => await apps.getApp(`${appName}@${appVersion}`).then(prop('id'))
+
+const injectedDependencies = async (builderName: string, builderVersion: string, workspace: string, account: string, environment: string) => {
+  const extension = (environment === 'prod') ? 'myvtex' : 'myvtexdev'  // Remove this
+  const http = axios.create({
+    baseURL: `https://${workspace}--${account}.${extension}.com`, // change this route!!!
+    timeout: builderHubInjectedDepsTimeout,
+  })
+  try {
+    const res = await http.get(`/_v/private/builder/0/getdeps/${builderName}/${builderVersion}`)
```

Comment:
*Idea*: Couldn't we make a single request with the app's `manifest.json`, in such a way that `builder-hub` returns all injected deps for all builders (in this case, it would continue to be only the `react` one).